hir/src/pass/manager.rs

```diff
@@ -845,6 +845,13 @@ impl OpToOpPassAdaptor {
         instrumentor: Option<Rc<PassInstrumentor>>,
         parent_info: Option<&PipelineParentInfo>,
     ) -> Result<(), Report> {
+
+        if verify {
```

Comment:
Would be great if this were an enum instead of a bool but it was there from before so it should be fine to leave it as-is.